pages/agent/v3/tokens.md

```diff
@@ -1,100 +1,126 @@
 
-You can find your `organization-id` in your Buildkite organization settings page, or by running the following GrapqQL query:
+### Using the REST API
 
-```graphql
-query GetOrgID {
-  organization(slug: "organization-slug") {
-    id
-  }
-}
+To [create an agent token](/docs/apis/rest-api/clusters#agent-tokens-create-a-token) using the [REST API](/docs/apis/rest-api), run the following example `curl` command:
+
+```curl
+curl -H "Authorization: Bearer $TOKEN" \
+  -X POST "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id}/tokens" \
+  -H "Content-Type: application/json" \
+  -d '{ "description": "A description" }'
 ```
 
+where:
+
+- The `$TOKEN` value is an [API access token](https://buildkite.com/user/api-access-tokens) scoped to the relevant _Organization_ and _REST API Scopes_ that your agent needs access to in Buildkite.
+
+- The `{org.slug}` value can be obtained:
+
+    * From the end of your Buildkite URL after accessing the _Pipelines_ page of your organization in Buildkite.
+
+    * By running the [List organizations](/docs/apis/rest-api/organizations#list-organizations) REST API query to obtain this value from `slug` in the response. For example:
+
+        ```curl
+        curl -H "Authorization: Bearer $TOKEN" "https://api.buildkite.com/v2/organizations"
+        ```
+
+- The `{cluster.id}` value can be obtained:
+
+    * From the _Cluster Settings_ page of your specific cluster that the agent will connect to. To do this:
+
+        - Select _Agents_ > the specific cluster > _Settings_.
+        - Once on the _Cluster Settings_ page, copy the `id` parameter value from the _GraphQL API Integration_ section, which is the `{cluster.id}` value.
+
+    * By running the [List clusters](/docs/apis/rest-api/clusters#clusters-list-clusters) REST API query and obtain this value from the `id` in the response associated with the name of your cluster (specified by the `name` value in the response). For example:
+
+        ```curl
+        curl -H "Authorization: Bearer $TOKEN" "https://api.buildkite.com/v2/organizations/{org.slug}/clusters"
+        ```
+
 <!--alex ignore clearly-->
 
-The token description should clearly identify the environment the token is intended to be used for, and is shown on your [Agents page](https://buildkite.com/organizations/-/agents) (for example, `Read-only token for static site generator`).  
+- The `description` value clearly identifies the environment the token is intended to be used for (for example, `Read-only token for static site generator`), and is listed on the _Agent tokens_ page of your specific cluster the agent connects to. To access this page, select _Agents_ > the specific cluster > _Agent Tokens_.
```

Comment:
Should this match the earlier one, as follows?

```suggestion
    **Note** The `description` value should clearly identifies the environment the token is intended to be used for (for example, `Read-only token for static site generator`), as it is listed on the _Agent tokens_ page of your specific cluster the agent connects to. To access this page, select _Agents_ > the specific cluster > _Agent Tokens_.
```

Reply:
This should probably be left as a bullet point in this case as it describes one of the options in a the `curl` command above it.
Thanks for pointing out the wording harmonisation with the description above for the steps described in the Buildkite interface equivalent for this process! I'll incorporate that for sure :-)